src/software/ai/hl/stp/play/enemy_free_kick_play_test.cpp

```diff
@@ -31,11 +31,21 @@ TEST_F(EnemyFreekickPlayTest, test_enemy_free_kick_play)
     setAIPlay(TYPENAME(EnemyFreekickPlay));
     setRefereeCommand(RefereeCommand::NORMAL_START, RefereeCommand::INDIRECT_FREE_THEM);
 
-    std::vector<ValidationFunction> terminating_validation_functions = {};
+    std::vector<ValidationFunction> terminating_validation_functions = {
+        // This will keep the test running for 9.5 seconds to give everything enough
+        // time to settle into position and be observed with the Visualizer
+        // TODO: Implement proper validation
```

Comment:
Should the TODO be removed since this is finished?

Reply:
this  TODO is for adding some real validation functions to test the behaviour